### DISCUSSION TITLE
Error Dialog

### DISCUSSION THREAD

**Discussion by catalin-pr** [mtex-toolbox/mtex#759] [Ask Anything] [answered]
Dear All,

Just a question !

When I try to load a new data as ctf. indicate the following error: EBSD format 'CTF' does not match the data file: "C:/Users/.../f.ctf"

Is there anybody which had this error and if so please can indicate from where is this error

Many thanks,
Catalin

**Comment by zmichels** [comment]
I have had this error before a couple times. One time, exporting the .CTF from Oxford software again solved the issue. Another time, that did not do the trick, and I had to use the .CRC+.CPR files (import the .CRC file), and that worked.

What is the provenance of the dataset?

Zach

**Comment by ralfHielscher** [comment]
You may also send the data set so we can investigate where the error
originates from.

Ralf.

********************************************************************
Ralf Hielscher                   Tel: +371-531-38556
Fakultät für Mathematik               +371-531-22200 (Sekr.)
Technische Universität Chemnitz  Fax: +371-531-22109
Reichenhainer Str. 39            E-mail:
ralf.hielscher@mathematik.tu-chemnitz.de
D-09126 Chemnitz                 http://www.tu-chemnitz.de/~rahi
********************************************************************

**Comment by catalin-pr** [comment]
[300.zip](https://github.com/mtex-toolbox/mtex/files/5954184/300.zip)

**Comment by catalin-pr** [comment]
Thanks for quick response, I have attached below it. It was saved from Esprit 2.1

**Comment by catalin-pr** [comment]
It is possible the Matlab version 2020b to be the problem as also in the calculation of another file (saved some times ago) that seems to work generate the following error: 
Error using curvatureTensor/fitDislocationSystems (line 30)
Optimization Toolbox not found. The function fitDislocationSystems depends on the Matlab Optimzation Toolbox
or, more specifically, on the function linprog.

**Reply by zmichels** [reply]
Here is a link about the Optimization Toolbox
https://www.mathworks.com/products/optimization.html

In the Matlab window/toolbar, you can click on the "Add-Ons" button/drop-down to try to add the package. Availability may depend on your Matlab license, I think.

**Reply by ralfHielscher** [reply]
This question is unrelated to the original post. Please use new threads for new questions.

Ralf.

**Comment by zmichels** [comment, marked as answer]
I replaced the commas with periods at decimal points and now it imports fine for me – I used a text editor. I tried to attach the edited file, but it was too large.

**Reply by ralfHielscher** [reply]
Yes, the commas are the problem. I adapted the ctf interface such that it can deal with this situation as well. See e5c99c6

This or replace commas by dots manually should do the trick.

Ralf.

**Reply by catalin-pr** [reply]
I saved the ctf files in another computer (using Esprit) and I runned in Matlab 2018 and now does work

**Comment by catalin-pr** [comment]
in past I used matlab 2019a

**Comment by zmichels** [comment]
The other error you mention is because the Optimization Toolbox is not installed in your Matlab… but it sounds like it was in your last version.

**Comment by zmichels** [comment]
I came on the browser version to upload the modified datafile I mentioned. I think uploading this route should work.
[300_ZM.zip](https://github.com/mtex-toolbox/mtex/files/5954282/300_ZM.zip)

**Reply by catalin-pr** [reply]
Thanks Zach, indeed I solved the issue using Optimization Toolbox 
Also your saved data works fine!
Now I am trying to see how I can save it without the editor because I am not using it and don't know how to use it

**Comment by catalin-pr** [comment]
Thanks Ralf

I will put in another post

**Comment by catalin-pr** [comment]
Thank you Ralf.

I will let you soon if I solved the issue.
